### PR TITLE
Add new LXC: Kibana

### PR DESCRIPTION
**Pull request by Elkozel**
## Description
This provides two new scripts for Kibana, widely used for system and security monitoring. The added scripts are:

- ct/kibana.sh
- install/kibana-install.sh

Description about the files are below.

### ct/kibana.sh
The file provides a typical installation, based on the `adguard.sh` installation script provided almost without any changes.

### install/kibana-install.sh
This script follows the `n8n-install.sh` script, completing almost the same steps for installation using `apt-get`. 

## Fixes
No issues were created for this improvement.

## Type of change

Please check the relevant option(s):

- [ ] Bug fix (non-breaking change that resolves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (a fix or feature that would cause existing functionality to change unexpectedly)
- [x] New script (a fully functional and thoroughly tested script or set of scripts.)
- [x] Self-review performed (I have reviewed my code, ensuring it follows established patterns and conventions)
- [x] Documentation update required (this change requires an update to the documentation)

### CODE REVIEW

File: ct/kibana.sh [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+msg_ok "Updated Successfully"
+exit
+}
+
+start
+build_container
+description
+
+msg_ok "Completed Successfully!\n"
+echo -e "${APP} is installed! For security purposes, ${APP} is only reachable from localhost.\n
```
Comment (tteck) [inline ct/kibana.sh:78]: not adhering to the established patterns and conventions

File: ct/kibana.sh [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+To access the ${APP} dashboard, please adjust ${BL}server.host${CL} in ${BL}/etc/kibana/kibana.yml${CL}.\n
```
Comment (tteck) [inline ct/kibana.sh:79]: why not do it in the script?

File: install/kibana-install.sh [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
```
Comment (tteck) [inline install/kibana-install.sh:17]: missing "core" dependencies.
not adhering to the established patterns and conventions

File: install/kibana-install.sh [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+$STD apt-get install -y apt-transport-https
+$STD apt-get install -y gnupg
+msg_ok "Installed Dependencies"
+
+msg_info "Setting up Elastic Repository"
+mkdir -p /etc/apt/keyrings
```
Comment (tteck) [inline install/kibana-install.sh:23]: you make a directory, but never use it

File: ct/kibana.sh (hunk 2 above)
Comment (Elkozel) [reply ct/kibana.sh:79]: I will make it as a prompt

File: install/kibana-install.sh (hunk 3 above)
Comment (Elkozel) [reply install/kibana-install.sh:17]: By "core" dependencies, could you elaborate a bit more. I see a lot of scripts having:
- curl
- sudo
- mc

File: ct/kibana.sh (hunk 2 above)
Comment (tteck) [reply ct/kibana.sh:79]: No prompt, just use `sed -i 's/#server.host: "localhost"/server.host: "0.0.0.0"/' /etc/kibana/kibana.yml`

**Comment by tteck** [comment]
Keep working

**Pull request closed without merging**